packages/pyolite-kernel/py/pyolite/pyolite/kernel.py

```diff
@@ -71,7 +71,8 @@ def complete(self, code, cursor_pos):
     async def run(self, code):
         self.interpreter._last_traceback = None
         exec_code = self.interpreter.transform_cell(code)
-        await _load_packages_from_imports(exec_code)
+        if self.is_complete(exec_code)["status"] not in ["incomplete", "invalid"]:
```

Comment:
I believe we shouldn't do an is_complete check in the run request. It might be handled differently in ipykernel.